source/server/listener_impl.cc

```diff
@@ -36,6 +36,8 @@
 #include "source/common/quic/udp_gso_batch_writer.h"
 #endif
 
+#include "absl/strings/str_replace.h"
```

Comment:
nit: not required here

Reply:
I will remove it. Still need some to find root cause of CI failure.

Reply:
Now `cluster.xdstp://test/envoy.config.cluster.v3.Cluster/foo-cluster/baz?xds.node.cluster=cluster_name&xds.node.id=node_name` would become `cluster.xdstp://test/envoy.config.cluster.v3.Cluster/foo-cluster/baz?xds.node.cluster=cluster_name&xds.node.id=node_name`, I update codes to only format "//".

Reply:
Now it formats as `cluster.xdstp_test/envoy.config.cluster.v3.Cluster/foo-cluster/baz?xds.node.cluster=cluster_name&xds.node.id=node_name`

Reply:
Sorry for the force push :(